docs/sphinx/source/reference/sql_commands/DQL/CASE.rst

```diff
@@ -0,0 +1,305 @@
+====
+CASE
+====
+
+.. _case:
+
+Returns a result based on conditional evaluation, similar to if-then-else logic.
+
+Syntax
+======
+
+.. raw:: html
+    :file: CASE.diagram.svg
+
+CASE expressions use conditional evaluation:
+
+**Searched CASE** (evaluates conditions):
+
+.. code-block:: sql
+
+    CASE
+        WHEN condition1 THEN result1
+        WHEN condition2 THEN result2
+        ...
+        ELSE default_result
+    END
+
+Parameters
+==========
+
+``condition``
+    A boolean expression evaluated for each WHEN clause.
+
+``result``
+    The value to return if the corresponding condition is TRUE. Can be any expression, column, or literal.
+
+``ELSE result`` (optional)
```

Comment:
Do the names of the parameters defined here correspond to the ones used in the code block above? If that is true, I believe it will be better to rename this one to `default_result`, maybe.